core/services/job/orm.go

```diff
@@ -230,9 +230,19 @@ func (o *orm) CreateJob(jb *Job, qopts ...pg.QOpt) error {
 			if jb.OCR2OracleSpec.TransmitterID.Valid {
 				switch jb.OCR2OracleSpec.Relay {
 				case relay.EVM:
-					_, err := o.keyStore.Eth().Get(jb.OCR2OracleSpec.TransmitterID.String)
-					if err != nil {
-						return errors.Wrapf(ErrNoSuchTransmitterKey, "%v", jb.OCR2OracleSpec.TransmitterID)
+					chains := o.chainSet.Chains()
+					var useForwarders = true
+					for _, c := range chains {
```

Comment:
Why do we have to check for all chains here? couldn't we get the chain from the spec and only check for that one chain?

Reply:
Changing this to use a `EVMChainForJob` function